src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/EnvelopeEventProcessor.java

```diff
@@ -98,6 +100,16 @@ private MessageProcessingResult process(IMessage message) {
                 // not repeat them, at least until CCD operations become idempotent
                 return new MessageProcessingResult(UNRECOVERABLE_FAILURE, ex);
             } catch (Exception ex) {
+                if (ex instanceof FeignException) {
```

Comment:
This new bit needs to be removed from this PR - I added that on a different branch for troubleshooting purposes, but we can't log full response content, as it might contain sensitive information.